orion/core.py

```diff
@@ -72,6 +72,20 @@ def __eq__(self, other):
             self._fitted == other._fitted
         )
 
+    def __repr__(self):
+        hyperparameters = None
+        if self._hyperparameters is not None:
+            hyperparameters = '\n'.join(
+                '    {}: {}'.format(step, value) for step, value in self._hyperparameters.items())
+
+        return (
+            'Orion({})\n'
+            'hyperparameters:\n{}\n'
+        ).format(
+            self._pipeline,
```

Comment:
Does the pipeline have its own string representation or does this just print the object reference?

Reply:
That is a good point. 

I just made an edit that shows the `primitives` in case the pipeline is an instance of an `MLPipeline`.
I think that's the best was to represent it, do you have another suggestion?

Reply:
I think that makes sense!